### ISSUE TITLE
Render selected keyframes doesn't seem to work

### ISSUE THREAD

**Issue by ann40** [p2or/blender-loom#25]
I selected some of my grease pencil drawing, and open the menu, but all of them are grey, and when I click on " render select keyframe" it doesn't respond at all.

![149209250-4a92ac81-3c61-4be8-9f64-6d872139f5f8](https://user-images.githubusercontent.com/512368/149329068-c988ead7-4481-45e5-8a12-d351e4b2070d.png)

**Comment by p2or** [comment]
Hi @ann40,

just tested with a fresh scene and can not reproduce your issue, all is working fine on Blender 3.0.0 and the latest release of loom, see:

<img width="1288" alt="Screenshot 2022-01-13 at 13 27 41" src="https://user-images.githubusercontent.com/512368/149330262-242d983e-d4f6-4910-8921-9ed25f0b34c6.png">

Do you mind sharing a simplified file to see what's going on?

Cheers,
Christian

**Comment by ann40** [comment]
Hi, Christian

I tried on a new file, yes you are right it's working on the new file, I am not sure if it's because that file was a file from version 2.90, so
it has some problem. anyway, I don't really need to do anything with that file, so it's fine for me now, thanks for the reply. But if you
are still interested in why that one doesn't work I attached the file here in the email, just in case .

thanks![https://outlook-2.cdn.office.net/assets/mail/file-icon/png/generic_16x16.png]jump_Practise.blend<https://1drv.ms/u/s!AiScswZ9LQ0Sl04pDmKOB-mkcS_j>
Ann

________________________________

**Comment by p2or** [comment]
Hi @ann40,

the output file format was set to FFmpeg, which is not supported. Change that setting to to any image file type (jpg, tif, exr...) and it works fine, see:

<img width="990" alt="Screenshot 2022-01-13 at 16 55 51" src="https://user-images.githubusercontent.com/512368/149364181-f326b272-66bd-4074-919a-ff1639c01692.png">

If you want a movie file, you would have to 'fill the missing frames' in the first place and then encode the image sequence to mp4, mov or whatever using [encoding dialog](documentation https://github.com/p2or/blender-loom#encode-image-sequence-dialog-ctrlshiftf9).

FYI: I just improved the keyframe selection for the 'Grease Pencil'-Editor. While playing with the ['greasepencil-bike.blend' demo](https://www.blender.org/download/demo-files/#grease-pencil) in the meantime, I've noticed that all keyframes are getting selected, even those that are not visible. Anyway, if you want to test it, make sure you download loom using the green 'Code'-button until the next realease.

Cheers,
Christian

**Comment by ann40** [comment]
I see, thanks for the quick response!On Jan 13, 2022 11:18 AM, p2or ***@***.***> wrote:
Hi @ann40,
the output file format was set to FFmpeg, which is not supported. Change that setting to to any image file type (jpg, tif, exr...) and it works fine, see:

If you want a movie file, you would have to 'fill the missing frames' in the first place and then encode the image sequence to mp4, mov or whatever using [encoding dialog](documentation https://github.com/p2or/blender-loom#encode-image-sequence-dialog-ctrlshiftf9).
FYI: I just improved the keyframe selection for the 'Grease Pencil'-Editor. While playing with the 'greasepencil-bike.blend' demo in the meantime, I've noticed that all keyframes are getting selected, even those that are not visible. Anyway, if you want to test it, make sure you download loom using the green 'Code'-button until the next realease.
Cheers,
Christian
